docs/pages/accounts/sso.mdx

```diff
@@ -6,7 +6,12 @@ description: Learn how your enterprise organization can use your Identity Provid
 import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
-SSO is available for [Enterprise plan](https://expo.dev/pricing) customers. To get started, [contact us](https://expo.dev/contact) and we'll help you set it up for your organization. You can also start preparing your identity provider for Expo SSO by following the configuration guides below.
+SSO is available for [Enterprise plan](https://expo.dev/pricing) customers and set up is self-service.
+
+To get started, first prepare your identity provider (IdP) for Expo SSO and gather information by following a configuration guide below.
+Once you have done this, the owner of your Organization can enable SSO, as described below.
```

Comment:
```suggestion
To get started, prepare your identity provider (IdP) for Expo SSO and gather information by following the [configuration guide for your IdP](/#identity-provider-support) from below. Once you have done this, your Organization's owner can follow instructions to [enable SSO](#enable-sso).
```

Let's rephrase this a little bit for clarity and use internal links to the sections.

Reply:
Done